src/Command/UpdateFormerAttributesCommand.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Command;
+
+use Akeneo\PimEnterprise\ApiClient\AkeneoPimEnterpriseClientInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class UpdateFormerAttributesCommand extends Command
+{
+    protected static $defaultName = 'app:update-former-attributes';
+
+    /** @var AkeneoPimEnterpriseClientInterface */
+    private $client;
+
+    /** @var string */
+    private $assetFamilyCode;
+
+    /** @var SymfonyStyle */
+    private $io;
+
+    public function __construct(AkeneoPimEnterpriseClientInterface $client)
+    {
+        parent::__construct($this::$defaultName);
+
+        $this->client = $client;
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setDescription('Update the former PIM attributes from PAM to Assets')
+            ->addArgument('assetFamilyCode', InputArgument::REQUIRED, 'The asset family code to link to')
+            ->addOption('attributeCodes', null, InputOption::VALUE_OPTIONAL, 'List of attribute codes to update the attributes')
+        ;
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output)
+    {
+        $attributeCodes = $input->getOption('attributeCodes');
+        if (null !== $attributeCodes) {
+            $attributeCodes = preg_split('/ *, */', $attributeCodes);
+        }
+
+        $this->io = new SymfonyStyle($input, $output);
+        $this->assetFamilyCode = $input->getArgument('assetFamilyCode');
+
+        $count = 0;
+        $attributes = $this->client->getAttributeApi()->all();
```

Comment:
Why don't you just fetch the attributes directly in DB  and filter on media_type ?